cyberdrop_dl/utils/sorting.py

```diff
@@ -159,10 +158,13 @@ async def sort_image(self, file: Path, base_name: str) -> None:
             return
         height = resolution = width = None
         try:
-            with Image.open(file) as image:
-                width, height = image.size
+            width, height = await asyncio.to_thread(imagesize.get, str(file))
```

Comment:
You can pass the `Path` object as is here
```suggestion
            width, height = await asyncio.to_thread(imagesize.get, file)
```